internal/boxcli/shellenv.go

```diff
@@ -40,6 +42,9 @@ func shellEnvCmd() *cobra.Command {
 	command.Flags().BoolVar(
 		&flags.install, "install", false, "install packages before exporting shell environment")
 
+	command.Flags().BoolVar(
+		&flags.pure, "pure", false, "If this flag is specified, devbox creates an isolated environment inheriting almost no variables from the current environment. A few variables, in particular HOME, USER and DISPLAY, are retained.")
```

Comment:
could make a `const pureFlagMsg = "...."` that is reused in all these places, instead of copy-pasta

Reply:
Nice idea! The description is slightly different for shell, run and shellenv. So I decided to copy each time for now. In a follow up try making a function that reuses the same variable but maybe differs in description.

Reply:
oh they're different? nevermind mind me then